src/main/java/de/tum/in/www1/artemis/service/GroupNotificationService.java

```diff
@@ -120,6 +120,16 @@ public void notifyInstructorGroupAboutDuplicateTestCasesForExercise(Exercise exe
         saveAndSend(createNotification(exercise, null, GroupNotificationType.INSTRUCTOR, NotificationType.DUPLICATE_TEST_CASE, notificationText));
     }
 
+    /**
+     * Notify instructor groups about invalid submissions
```

Comment:
```suggestion
     * Notify instructor groups about illegal submissions
```

Reply:
please be more specific

Reply:
the text should also be more detailed please

Reply:
done in 2aef655083373af302f175c973b2e466b5c230a1